score-lang-cli/src/main/java/org/openscore/lang/cli/services/ScoreServices.java

```diff
@@ -24,4 +24,5 @@
     void subscribe(ScoreEventListener eventHandler, Set<String> eventTypes);
     Long trigger(CompilationArtifact compilationArtifact, Map<String, ? extends Serializable> inputs, Map<String, ? extends Serializable> systemProperties);
     Long triggerSync(CompilationArtifact compilationArtifact, Map<String, ? extends Serializable> inputs, Map<String, ? extends Serializable> systemProperties);
+    Long triggerSync(CompilationArtifact compilationArtifact, Map<String, ? extends Serializable> inputs, Map<String, ? extends Serializable> systemProperties, boolean quiet);
```

Comment:
I meant to delete the old one. I didn't have time to check if everything still works fine so I overloaded the method.